docs/deployment/allocation.md

```diff
@@ -63,6 +63,21 @@ creating a new allocation queue:
 - `--backlog <count>` How many allocations should be queued (waiting to be started) in PBS/Slurm at any given time.
 - `--workers-per-alloc <count>` How many nodes should be requested in each allocation.
 - `--name <name>` Name of the allocation queue. Will be used to name allocations. Serves for debug purposes only.
+- **Worker resources** You can specify [CPU](../jobs/cresources.md) and [generic](../jobs/gresources.md)
```

Comment:
Maybe explicitly mention that otherwise the worker will detect resources?

Reply:
I agree, I added it to docs.